notebooks/Sigma_test_Notebook.ipynb

```diff
@@ -0,0 +1,340 @@
+{
+ "metadata": {
+  "language_info": {
+   "codemirror_mode": {
+    "name": "ipython",
+    "version": 3
+   },
+   "file_extension": ".py",
+   "mimetype": "text/x-python",
+   "name": "python",
+   "nbconvert_exporter": "python",
+   "pygments_lexer": "ipython3",
+   "version": "3.8.5-final"
+  },
+  "orig_nbformat": 2,
+  "kernelspec": {
+   "name": "python3",
+   "display_name": "Python 3.8.5 64-bit ('.venv')",
+   "metadata": {
+    "interpreter": {
+     "hash": "74f679a4fc854d75b381693882d369b7543a476b4b9cfda76eebb965e9131c3c"
+    }
+   }
+  }
+ },
+ "nbformat": 4,
+ "nbformat_minor": 2,
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#!pip install -q timesketch_api_client"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "pip install ../api_client/python"
```

Comment:
ok... so why the `#pip install` before and now `pip install` from a path?

Remove this cell, uncomment the other one, we should be wanting to install the pypi version.

Reply:
you are also not installing `picatrix`, are you making the assumption you are running a picatrix shell? If so, you've already got the timesketch api client installed, since it's a requirement... if not, then I suggest you also install picatrix

Reply:
have not used picatrix in that example

Reply:
no, I don't see any picatrix references, I was just. wasn't sure what `requirements` would be here,  but if there are no dependencies on either picatrix nor the api client, then just having the `pip install` of hte api client is fine,